lib/rpcServer/commands/platform/getDataContractHandlerFactory.js

```diff
@@ -0,0 +1,60 @@
+const {
+  server: {
+    error: {
+      InvalidArgumentGrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+
+const RPCError = require('../../../rpcServer/RPCError');
+const Validator = require('../../../utils/Validator');
+const argsSchema = require('./schemas/getDataContract');
+const ValidationError = require('../../../errors/ArgumentsValidationError');
+
+const validator = new Validator(argsSchema);
+
+/**
+ *
+ * @param {DriveAdapter} driveAPI
+ * @param {DashPlatformProtocol} dpp
+ * @returns {getDataContractHandler}
+ */
+function getDataContractHandlerFactory(driveAPI, dpp) {
+  /**
+   * @typedef getDataContractHandler
+   * @param {Object} args
+   * @param {string} args.id - contract id
+   * @returns {Promise<{ dataContract: string }>}
+   */
+  async function getDataContractHandler(args) {
+    validator.validate(args);
+    const { id } = args;
+
+    if (id == null) {
```

Comment:
=== ?

Reply:
It's a conventional method to check for null and undefined at the same time